code/game/machinery/vending.dm

```diff
@@ -1403,11 +1403,23 @@
 	product_ads = "Praise!;Pray!;Obey!"
 	icon_state = "teomat"
 	vendor_department = DEPARTMENT_CHURCH
-	products = list(/obj/item/book/ritual/cruciform = 10, /obj/item/storage/fancy/candle_box = 10, /obj/item/reagent_containers/food/drinks/bottle/ntcahors = 20)
+	products = list(
+		/obj/item/book/ritual/cruciform = 10, 
+		/obj/item/storage/fancy/candle_box = 10, 
+		/obj/item/reagent_containers/food/drinks/bottle/ntcahors = 20, 
+		/obj/item/clothing/accessory/job/cape/church/small = 5, 
+		/obj/item/clothing/accessory/job/cape/church/small/red = 5,
+		/obj/item/clothing/accessory/job/cape/church/small/white = 5
+		)
 	contraband = list(/obj/item/implant/core_implant/cruciform = 3)
-	prices = list(/obj/item/book/ritual/cruciform = 500, /obj/item/storage/fancy/candle_box = 200, /obj/item/reagent_containers/food/drinks/bottle/ntcahors = 250,
-				/obj/item/implant/core_implant/cruciform = 1000)
-
+	prices = list(
+		/obj/item/book/ritual/cruciform = 500, 
+		/obj/item/storage/fancy/candle_box = 200, 
+		/obj/item/reagent_containers/food/drinks/bottle/ntcahors = 250, 
+		/obj/item/clothing/accessory/job/cape/church/small = 200, 
+		/obj/item/clothing/accessory/job/cape/church/small/red = 200,
+		/obj/item/clothing/accessory/job/cape/church/small/white = 200,
+	)
```

Comment:
You may have accidentally removed the price listing for `/obj/item/implant/core_implant/cruciform` here.

Reply:
That did indeed happen, and will be fixed